conans/client/toolchain/make.py

```diff
@@ -22,172 +16,26 @@ class MakeToolchain(object):
         ifndef CONAN_TOOLCHAIN_INCLUDED
             CONAN_TOOLCHAIN_INCLUDED = TRUE
 
-            # Automatic Conan Toolchain Variables
-        {%- if build_type %}
-            CONAN_TC_BUILD_TYPE = {{build_type}}
-        {%- endif -%}
-        {%- if os_host %}
-            CONAN_TC_OS_HOST = {{os_host}}
-        {%- endif %}
-        {%- if arch_host %}
-            CONAN_TC_ARCH_HOST = {{arch_host}}
-        {%- endif %}
-        {%- if triplet_host %}
-            CONAN_TC_TRIPLET_HOST = {{triplet_host}}
-        {%- endif %}
-        {%- if os_build %}
-            CONAN_TC_OS_BUILD = {{os_build}}
-        {%- endif %}
-        {%- if arch_build %}
-            CONAN_TC_ARCH_BUILD = {{arch_build}}
-        {%- endif %}
-        {%- if triplet_build %}
-            CONAN_TC_TRIPLET_BUILD = {{triplet_build}}
-        {%- endif %}
-        {%- if os_target %}
-            CONAN_TC_OS_TARGET = {{os_target}}
-        {%- endif %}
-        {%- if arch_target %}
-            CONAN_TC_ARCH_TARGET = {{arch_target}}
-        {%- endif %}
-        {%- if triplet_target %}
-            CONAN_TC_TRIPLET_TARGET = {{triplet_target}}
-        {%- endif %}
-        {%- if compiler %}
-            CONAN_TC_COMPILER = {{compiler}}
-        {%- endif %}
-        {%- if compiler_version %}
-            CONAN_TC_COMPILER_VERSION = {{compiler_version}}
-        {%- endif %}
-        {%- if compiler_runtime %}
-            CONAN_TC_COMPILER_RUNTIME = {{compiler_runtime}}
-        {%- endif %}
-
             # Recipe-Defined Variables
         {%- for it, value in variables.items() %}
             {{ it }} = {{ value }}
         {%- endfor %}
 
-            # Automatic Conan pre-processor definition: build_type_define
-        {%- if build_type_define %}
-            CONAN_TC_CPPFLAGS += -D{{build_type_define}}
```

Comment:
But this was being used, it was already part of the build

Reply:
restored